Mono.Debugging.Soft/PortablePdbData.cs

```diff
@@ -63,6 +67,7 @@ public static bool IsPortablePdb (string pdbFileName)
 		public PortablePdbData (string pdbFileName)
 		{
 			this.pdbFileName = pdbFileName;
+			sourceLinkMaps = new Lazy<IEnumerable<SourceLinkMap>> (GetSourceLinkMaps);
```

Comment:
Can we use SourceLinkMap[] instead of IEnumerable?

Reply:
IEnumerable means lazy evaluation for me, and we have the data ready.

Reply:
Changed.